src/Api.js

```diff
@@ -31,6 +27,17 @@ var Api = {
     }
   },
 
+  flightEndpoints: {
```

Comment:
Is this only gonna be use here? can we make it private by prepending `_` (i.e. `_flightEndpoints`)
I think it would be the same for `hotelEndpoints`

Reply:
This is also optional, just to avoid confusion between public/private